src/net/sf/memoranda/ui/PSPControlPanel.java

```diff
@@ -0,0 +1,39 @@
+package net.sf.memoranda.ui;
```

Comment:
Also, according to our new coding standards, this class needs a few banner comments (for file, class, and methods).